go/ct/st/block_context.go

```diff
@@ -32,6 +34,44 @@ type BlockContext struct {
 	TimeStamp   uint64     // Block's timestamp in unix time in seconds
 }
 
+// NewBlockContext creates a new block context
+func NewBlockContext() *BlockContext {
+	return &BlockContext{BlobHashes: []vm.Hash{}}
```

Comment:
The handling of the `BlockContext` could be simplified if `nil` is considered a valid value for the list.